README.md

```diff
@@ -124,39 +124,6 @@ Environment variables:
 
 A basic Docker image where we can run a `curl` command to bootstrap the environment with data from `demo_data.json`
 
```

Comment:
Check the `idp` section in the README too to ensure it is still accurate.

Also, I think it would be worthwhile to add a section for the `idp` and `ldap` configurations to state that they should only be used for testing, they are not meant for production use.